panda/src/navigation/config_navigation.cxx

```diff
@@ -0,0 +1,53 @@
+/**
+ * PANDA 3D SOFTWARE
+ * Copyright (c) Carnegie Mellon University.  All rights reserved.
+ *
+ * All use of this software is subject to the terms of the revised BSD
+ * license.  You should have received a copy of this license along
+ * with this source code in a file named "LICENSE."
+ *
+ * @file config_navigation.cxx
+ * @author ashwini
+ * @date 2020-060-21
+ */
+
+#include "config_navigation.h"
+
+#include "pandaSystem.h"
+#include "dconfig.h"
+#include "navMesh.h"
+#include "navMeshBuilder.h"
+#include "navMeshNode.h"
+#include "navMeshQuery.h"
+
+#if !defined(CPPPARSER) && !defined(LINK_ALL_STATIC) && !defined(BUILDING_RECASTDETOUR)
+  #error Buildsystem error: BUILDING_NAVIGATION not defined
+#endif
+
+Configure(config_navigation);
+NotifyCategoryDef(navigation, "");
+
+ConfigureFn(config_navigation) {
+  init_libnavigation();
+}
+
+ConfigVariableInt navigation_sample_config_variable
+("navigation-sample-config-variable", 25);
```

Comment:
Do not add 'config-variable' to the name, and please add a description.

Reply:
If this is not used for anything it should be removed.

Reply:
Removed